ldes-server-port-publication-rest/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/constants/PublicationRestTestConstants.java

```diff
@@ -0,0 +1,12 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.rest.constants;
+
+public final class PublicationRestTestConstants {
+
+    public static final String FRAGMENTATION_VALUE_1 = "2020-12-28T09:36:09.72Z";
+
+     private PublicationRestTestConstants() {
+        // No need to instantiate the class, since there are only constants in it
```

Comment:
Although the comment clarifies it, I'd delete it since it clutters the code.

Reply:
I think that will address all your comments :-D